easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-9-foss-2021a.eb

```diff
@@ -0,0 +1,38 @@
+name = 'OpenFOAM'
+version = '9'
+
+homepage = 'https://www.openfoam.org/'
+description = """OpenFOAM is a free, open source CFD software package.
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer,
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+# toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/OpenFOAM/OpenFOAM-%(version_major)s/archive']
+sources = ['version-%(version)s.tar.gz']
+patches = ['OpenFOAM-%(version)s-cleanup.patch']
```

Comment:
I forgot to include the patch file, yet the test suite still passed?!

Reply:
The test suite failed, but we did not get the failure - from https://github.com/easybuilders/easybuild-easyconfigs/runs/3363235309#step:8:131:
```
2021-08-18T16:22:50.8554539Z ======================================================================
2021-08-18T16:22:50.8557882Z FAIL: test__parse_easyconfig_OpenFOAM-9-foss-2021a.eb (test.easyconfigs.easyconfigs.EasyConfigTest)
2021-08-18T16:22:50.8560934Z Test for easyconfig OpenFOAM-9-foss-2021a.eb
2021-08-18T16:22:50.8562783Z ----------------------------------------------------------------------
2021-08-18T16:22:50.8563401Z Traceback (most recent call last):
2021-08-18T16:22:50.8565235Z   File "test/easyconfigs/easyconfigs.py", line 1348, in innertest
2021-08-18T16:22:50.8566038Z     template_easyconfig_test(self, spec_path)
2021-08-18T16:22:50.8566947Z   File "test/easyconfigs/easyconfigs.py", line 1223, in template_easyconfig_test
2021-08-18T16:22:50.8567826Z     self.assertTrue(os.path.isfile(patch_full), msg)
2021-08-18T16:22:50.8569643Z AssertionError: Patch file /home/runner/work/easybuild-easyconfigs/easybuild-easyconfigs/easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-9-cleanup.patch is available for OpenFOAM-9-foss-2021a.eb
2021-08-18T16:22:50.8570775Z 
2021-08-18T16:22:50.8571354Z ----------------------------------------------------------------------
2021-08-18T16:22:50.8571960Z Ran 13091 tests in 505.157s
2021-08-18T16:22:50.8572208Z 
2021-08-18T16:22:50.8572524Z FAILED (failures=1)
```

Reply:
Broken by #13745 - I've add a revert PR in #13770